src/Specifications.md

```diff
@@ -1,17 +1,71 @@
 # Specifications
 
- &#xfeff;       | Game Boy                       | Game Boy Pocket | Game Boy Color  | Super Game Boy |
-----------------|--------------------------------|-----------------|-----------------|----------------|
-CPU             | 8-bit Sharp LR35902            |                 |                 |                |
-Clock           | 4.194304 MHz                   |                 | 8.388608 MHz    | 4.295454MHz (SGB1, NTSC)|
-Work RAM        | 8 KB                           |                 | 32 KB           |                |
-Video RAM       | 8 KB                           |                 | 16 KB           |                |
-Screen Size     | LCD 4,7 x 4,3 cm               | LCD 4,8 x 4,4 cm| TFT 4,4 x 4 cm  |                |
-Resolution      | 160x144                        |                 |                 | 256x224 (including border)|
-Sprites         | Max 40 per screen, 10 per line |                 |                 |                |
-Palettes        | 1x4 BG, 2x3 OBJ                |                 | 8x4 BG, 8x3 OBJ | 1+4x3, 4x15 (border) |
-Colors          | 4 grayshades                   |                 | 32768 colors    |                |
-Horizontal Sync | 9.198 KHz                      |                 |                 | 9.41986 KHz    |
-Vertical Sync   | 59.73 Hz                       |                 |                 | 61.1679 Hz     |
-Sound           | 4 channels with stereo sound   |                 |                 |                |
-Power           | DC6V 0.7W                      | DC3V 0.7W       | DC3V 0.6W       |                |
+<style>
+td {
+    text-align: center;
+}
+td:first-child {
+    text-align: left;
+}
+</style>
+
+<table>
+  <thead>
+    <tr>
+      <th></th><th>Game Boy (DMG)</th><th>Game Boy Pocket (MGB)</th><th>Super Game Boy (SGB)</th><th>Game Boy Color (CGB)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>CPU</td><td colspan="4">8-bit 8080-like Sharp CPU (speculated to be a SM83 core)</td>
+    </tr>
+    <tr>
+      <td>CPU freq</td><td colspan="2">4.194304&nbsp;MHz</td><td>Depends on model<sup class="footnote-reference"><a href="#sgb_clk">1</a></sup></td><td>Up to 8.388608&nbsp;MHz</td>
```

Comment:
Isn't Super Game Boy (SGB) the model? Are you referring to "revision"?

Reply:
Yep, right.